### DISCUSSION TITLE
how much time should the blockchain be behind current node time ?

### DISCUSSION THREAD

**Discussion by cryptosven76** [Chia-Network/chia-blockchain#8786] [Q&A]
i have a question on the delay of blockchain impact on farming chia. 
as you can see below my nodes time is 18:52:35 , but blockchain time is shown as 18:51:17 , which is more than a minute behind. is this expected ? 
it shows full sync, but i wonder if being that much behind can prevent me from successful farming ? 

root@node:/chia-blockchain# date ; chia show -s ; date
Sat Oct  9 18:52:35 MDT 2021
Current Blockchain Status: Full Node Synced

Peak: Hash: 27f8f2f9caee946aed3708cae915a07e3bce43ee81dee1759f0aa0095e9a859a
      Time: Sat Oct 09 2021 18:51:17 MDT                  Height:     975831

Estimated network space: 34.749 EiB
Current difficulty: 3008
Current VDF sub_slot_iters: 136314880
Total iterations since the start of the blockchain: 3590157855346

  Height: |   Hash:
   975831 | 27f8f2f9caee946aed3708cae915a07e3bce43ee81dee1759f0aa0095e9a859a
   975830 | f8783ae5a231636eb735427110533274cefde991bfe06f33bed6dad810245801
   975829 | 2cc1ca2e3311cac8502cc57b20b79cdf4fbaf7007022066e262e10327559c19a
   975828 | fc441ceb86c4529d3c819d1506d65a82fab68a88e0ffda38479061a35e36b3e8
   975827 | 6a4ff3c935632ae88a02ee2e3566b0546f641e3f85f7f22a476f7aaa579477e2
   975826 | e4d1f8e218773bd4c706898a5478cdfb1941398cd553dd4832eb5c76d8bdccff
   975825 | 7712421ca31dfcabb4bb934657b998cc21877c469d759f560f6629af91cfd2d2
   975824 | 21d494f97b343ea4cfc6dcc5fbdf725f574b0bbf83db15fbb5dac892cabd99a2
   975823 | bde9bf64dc3904df458d5d4a82e271cf7d9547b1c384a01a7656a170f8c54cf0
   975822 | cb4ee2417dfbac46300e9b3c8e277049b74398e8b3a39918c871af461909b506
Sat Oct  9 18:52:36 MDT 2021

**Comment by BrandtH22** [comment]
@cryptosven76 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note the previous transaction block is expected to be behind real time by upwards of a few minutes. This is because not all blocks are transactions blocks and those that are need to be infused into the chain while using the timestamp from their creation.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
